src/components/config/validate.ts

```diff
@@ -118,6 +118,11 @@ const PUSHOVER_NO_USER = setInvalidResponse('USER not found')
 const GOTIFY_NO_TOKEN = setInvalidResponse('TOKEN not found')
 const GOTIFY_NO_URL = setInvalidResponse('URL not found')
 
+// Pushbullet
+const PUSHBULLET_NO_TOKEN = setInvalidResponse(
+  'Token not found! You can create your Access Token at https://www.pushbullet.com/#settings'
```

Comment:
Maybe better say that it's "Pushbullet token not found!" to make it clearer.

Reply:
Fixed